docs/topics/migrations.txt

```diff
@@ -333,6 +336,12 @@ Note that this only works given two things:
   that your database doesn't match your models, you'll just get errors when
   migrations try to modify those tables.
 
+Note: if you have upgraded from Django 1.7 or before, and get
+'foreign key constraint' errors (for example, when you run manage.py test),
```

Comment:
I'm not sure about this as running tests with `manage.py test` shouldn't be affected by the migrations that are applied in the production database.

Also, these types of "upgrade" notes are better suited to the release notes.

Do you know if the root cause might be mixing apps with and without migrations? (https://code.djangoproject.com/ticket/24311)